wagtail_localize/wagtail_hooks.py

```diff
@@ -283,6 +297,48 @@ def register_restart_translation_page_action_menu_item():
     return RestartTranslationPageActionMenuItem(order=0)
 
 
+class ConvertToAliasPageActionMenuItem(PageActionMenuItem):
+    label = gettext_lazy("Convert to alias page")
+    name = "localize-convert-to-alias"
+    icon_name = "wagtail-localize-convert"
+    classname = "action-secondary"
+
+    def _is_shown(self, context):
+        # Only show this menu item on the edit view where there was a previous translation record
+        if context["view"] != "edit":
+            return False
+        page = context["page"]
+        try:
+            return (
+                page.alias_of_id is None
+                and Page.objects.filter(
```

Comment:
note: considered adding an additional check on `PageLogEntry.objects.get(page=page, action="wagtail.create_alias") but that felt like overkill